tests/test_datasets/test_transforms/test_formatting.py

```diff
@@ -130,3 +130,53 @@ def test_repr(self):
         cfg = dict(type='Collect', keys=['img'])
         transform = TRANSFORMS.build(cfg)
         self.assertEqual(repr(transform), "Collect(keys=['img'])")
+
+
+class FormatMultiTaskLabels(unittest.TestCase):
+
+    def test_transform(self):
+        img_path = osp.join(osp.dirname(__file__), '../../data/color.jpg')
+        data = {
+            'sample_idx': 1,
+            'img_path': img_path,
+            'ori_shape': (300, 400),
+            'img_shape': (300, 400),
+            'scale_factor': 1.0,
+            'flip': False,
+            'img': mmcv.imread(img_path),
+            'task1': 1,
+            'task3': 3,
+        }
+
+        cfg = dict(
+            type='FormatMultiTaskLabels',
+            tasks=['task1', 'task2', 'task3']
```

Comment:
is it mandatory to have the list of tasks here ? Can we do without it ?

Reply:
I can't find a way to retrieve the labels of the results without the names of the tasks